identity/app/model/CountryCodes.scala

```diff
@@ -2,242 +2,14 @@ package model
 
 object CountryCodes {
```

Comment:
Perhaps this should be called something like `PhoneNumbers` to make it clear what the codes are?